File: src/runtime/ts/std/time_native.ts
// Generated std/time.ts delegates host bindings through this native seam.

declare const process: { hrtime: { bigint(): bigint } };

export function perf_counter(): number {
    return Number(process.hrtime.bigint()) / 1_000_000_000;
}

export const perfCounter = perf_counter;

File: src/runtime/kotlin/std/time_native.kt
// Generated std/time.kt delegates host bindings through this native seam.

private val time_native_start = System.nanoTime()

fun time_native_perf_counter(): Double {
    return (System.nanoTime() - time_native_start).toDouble() / 1_000_000_000.0
}
